.terraform-docs.yaml

```diff
@@ -44,7 +44,7 @@ content: |-
     clumio = clumio
     }
     source                = "../../"
-    version               = "0.10.0"
+    version               = "0.17.0"
```

Comment:
Line 30 notes `... has version 0.10.0 ...` but this is `0.17.0`. Also should we use 0.19.0 here now?